openblas/riscv64.patch

```diff
@@ -0,0 +1,29 @@
+--- PKGBUILD.orig	2021-11-29 21:51:35.484915389 +0800
++++ PKGBUILD	2021-11-29 21:40:38.559150178 +0800
+@@ -6,7 +6,7 @@
+ pkgver=0.3.18
+ pkgrel=1
+ pkgdesc="An optimized BLAS library based on GotoBLAS2 1.13 BSD"
+ url="https://www.openblas.net/"
+ license=('BSD')
+ depends=('gcc-libs')
+@@ -20,9 +20,16 @@
+ build() {
+   cd $_pkgname-$pkgver
+ 
+-  make NO_STATIC=1 NO_LAPACK=1 NO_LAPACKE=1 NO_CBLAS=1 NO_AFFINITY=1 USE_OPENMP=1 \
```

Comment:
We can skip the `$CARCH` branches for now, because patching only the TARGET would actually make the change more clear. As you can see below.